### PR TITLE
Port bar drinks from tg

### PR DESCRIPTION
**Pull request by efzapa**
## About the PR 
This would add many drinks from tg straight into SS14, many of which we already have as functional reagents and drinks, but with no recipes.
It would also add some amount of preliminary mixing components needed to make drinks, like Sol Dry, Grenadine, Bitters, or Champagne
**Screenshots**

**Changelog**

:cl: gl0ck
- add: Added 20 new drinks for you to have fun with.

### CODE REVIEW

**Comment by efzapa** [comment]
@mirrorcult any opinions?

File: Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml [hunk 1]
```diff
@@ -319,6 +346,15 @@
   color: "#360000"
   spritePath: blackrussianglass.rsi
 
+- type: reagent
+  id: BlankPaper
+  name: blank paper
+  parent: BaseAlcohol
+  desc: A fizzy cocktail for those looking to start fresh. #This should in the future be able to reinstiute the oath of silence within a mime.
```
Comment (Veritius) [inline Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml:353]: I can code this if you want

Comment (efzapa) [reply Resources/Prototypes/Reagents/Consumable/Drink/alcohol.yml:353]: go ahead, I would but I have no knowledge on C# atm lmao

**Review by Veritius — APPROVED** [review]
lgtm

**Comment by Veritius** [comment]
though, shouldn't be merged without the blank slate functionality imo

File: Resources/Textures/Objects/Consumable/Drinks/amarettoglass.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/tgstation/tgstation/blob/master/icons/obj/drinks.dmi?raw=true", "states": [{"name": "icon"}]}
```
Comment (ZeroDayDaemon) [inline Resources/Textures/Objects/Consumable/Drinks/amarettoglass.rsi/meta.json:1]: All of these need permalinks

**Review by ZeroDayDaemon — APPROVED** [review]

File: Resources/Textures/Objects/Consumable/Drinks/amarettoglass.rsi/meta.json (hunk 2 above)
Comment (efzapa) [reply Resources/Textures/Objects/Consumable/Drinks/amarettoglass.rsi/meta.json:1]: can do once I get home 👍

**Comment by efzapa** [comment]
welp, #7398 seems to have fuggbutted my branch and i can't merge automatically, push to github, or anything really

gonna put on hold till i can figure out lol

**Comment by efzapa** [comment]
turns out git did a funny and pushed my upstream merges so all the files are fucked

git merged some funky changes from https://github.com/space-wizards/space-station-14/pull/8026/commits/079ae48b1d207d36572f34f0d2952806289462a3 and as a result, that branch is no longer building due to submodule fuckery

**Pull request closed without merging**